src/reference/asciidoc/whats-new.adoc

```diff
@@ -1,229 +1,13 @@
 [[whats-new]]
 === What's New
 
-==== Changes in 1.6 Since 1.5
+==== Changes in 2.6 Since 1.5
```

Comment:
Typo: in *2.0* :smile: 

Also I wonder if you are going to address XSD moving to `2.0` as a part of this PR as well .